src/main.c

```diff
@@ -173,6 +217,63 @@ void WriteInterfaces()
     free(InterfaceArray);
 }
 
+inline int
+CombineMetadataWithPacket(
+    _In_ HANDLE File,
+    _In_ PCHAR FragBuf,
+    _In_ unsigned long FragLength,
+    _In_ long InterfaceId,
+    _In_ long IsSend,
+    _In_ long TimeStampHigh, // usec (unless if_tsresol is used)
+    _In_ long TimeStampLow,
+    _In_ PDOT11_EXTSTA_RECV_CONTEXT Metadata,
+    _In_ unsigned long ProcessId
+)
+{
+
+    char Comment[MAX_PACKET_SIZE] = { 0 };
+    size_t commentlength = 0;
+
+    HRESULT Err = StringCchPrintfA((char*)&Comment, MAX_PACKET_SIZE, "Packet Metadata: ReceiveFlags:0x%x, PhyType:%s, CenterCh:%u, NumMPDUsReceived:%u, RSSI:%d, DataRate:%u, PID=%d",
+        Metadata->uReceiveFlags,
+        DOT11_PHY_TYPE_NAMES[Metadata->uPhyId],
+        Metadata->uChCenterFrequency,
+        Metadata->usNumberOfMPDUsReceived,
+        Metadata->lRSSI,
+        Metadata->ucDataRate,
+        ProcessId);
+
+    if (FAILED(Err))
+    {
```

Comment:
Fix to use K&R braces throughout PR.

Reply:
Nod, ok. For clarity, you only mean the bracing style yes? Some online are also suggesting/implying that with K&R also comes dropping braces on single-line code following conditionals, but I have always considered dropping the braces as risky.

Reply:
Oh man, you're right, they do omit braces for single-line code blocks in K&R. I agree that braces should still be used in that case. When I say "K&R" I really just mean 
if (x) {
} else {
}

Reply:
Got it, easy enough.